client/python/cryoet_data_portal/tests/test_get_client.py

```diff
@@ -6,6 +6,12 @@ def test_relationships(client) -> None:
     ds = datasets.pop()
     assert ds
     assert ds.authors.pop()
+    depo = ds.deposition
```

Comment:
I modified these tests to account for depositions and deposition authors in a way that seems consistent with what came before.

I think these could generally be more meaningful, but this may not be the right time to think too deeply. Open to ideas though.